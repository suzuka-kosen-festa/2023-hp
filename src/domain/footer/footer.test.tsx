import { composeStory } from "@storybook/react";
import { render } from "@testing-library/react";
import Meta, { Default } from "./footer.stories";

describe("[domain/footer] footer.tsx", () => {
  it("should render the Footer component", () => {
    const Composed = composeStory(Default, Meta);
    const { container } = render(<Composed />);
    expect(container).toMatchSnapshot();
  });
});
