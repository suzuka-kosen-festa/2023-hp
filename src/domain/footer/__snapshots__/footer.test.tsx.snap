// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`[domain/footer] footer.tsx > should render the Footer component 1`] = `
<div>
  <footer
    class="styles_footer__1rq13180"
  >
    <p
      class="styles_p__1rq13181"
    >
      © 
      第58回鈴鹿高専祭 実行委員会
    </p>
  </footer>
</div>
`;
